lib/deployment/preparation/preparation.spec.js

```diff
@@ -21,7 +21,28 @@ describe('Preparation', () => {
   });
 
   describe('on master branch', () => {
-    const options = { directory: 'source', branch: 'master' };
+    const options = { directory: 'source', branch: 'master', defaultBranch: 'master' };
+
+    it('does not create branch directory', () => {
+      expect(shell.mkdir).not.toBeCalled();
+      prepareDeployDirectory('deploy-directory', options);
+      expect(shell.mkdir).not.toBeCalled();
+    });
+
+    it('removes deploy directory to allow copying with replacement', () => {
+      prepareDeployDirectory('deploy-directory', options);
+      expect(shell.rm).toBeCalledWith('-rf', 'deploy-directory');
+    });
+
+    it('moves content from source directory to deploy directory', () => {
+      expect(shell.cp).not.toBeCalled();
+      prepareDeployDirectory('deploy-directory', options);
+      expect(shell.cp).toBeCalledWith('-r', 'source', 'deploy-directory');
+    });
+  });
+
+  describe('on a non-master branch with defaultbranch = non-master', () => {
```

Comment:
Hmm, what's the difference in the assertions between this and the previous `describe` block?

Reply:
Should we only keep the test for the changed behaviour?

Reply:
This is checking that defaultBranch gets assigned and works as expected but can be removed